src/control/content/components/LocationForm.js

```diff
@@ -14,14 +14,16 @@ class LocationForm extends React.Component {
 
   componentWillMount() {
     let model = {
+      id: '',
       title: '',
       subtitle: '',
       description: '',
       address: {},
       image: '',
       actionItems: [],
       categories: [],
-      carousel: []
+      carousel: [],
+      deeplinkUrl: ''
```

Comment:
remove carousel

Reply:
Carousel is used in the code, I'm just added comma to the end of line